src/webgpu/api/validation/state/device_lost/destroy.spec.ts

```diff
@@ -52,16 +53,15 @@ class DeviceDestroyTests extends AllFeaturesMaxLimitsValidationTest {
    * `fn` after the device is destroyed without any specific expectation. If `awaitLost` is true, we
    * also wait for device.lost to resolve before executing `fn` in the destroy case.
    */
-  async executeAfterDestroy(fn: () => void, awaitLost: boolean): Promise<void> {
+  async executeAfterDestroy(fn: () => void | Promise<void>, awaitLost: boolean): Promise<void> {
     this.expectDeviceLost('destroyed');
 
-    this.expectValidationError(fn, false);
```

Comment:
> Also fix a bug where some test code was running twice unintentionally.

fn is already called below, it should be called twice. We don't need to capture validation errors anymore because there are never any (and test teardown will check this). (Likely when this test was written, sometimes there were validation errors here.)